.github/workflows/cargo.yml

```diff
@@ -0,0 +1,11 @@
+name: cargo
+env:
+  CARGO_TERM_COLOR: always
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

Comment:
`v4` is the latest major version of `actions/checkout` at the time of this writing.